modules/statics/src/coins.ts

```diff
@@ -252,6 +252,7 @@ const GENERIC_TOKEN_FEATURES = [
 const TON_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.TSS, CoinFeature.TSS_COLD, CoinFeature.STAKING];
 const ARBETH_FEATURES = [...ETH_FEATURES, CoinFeature.MULTISIG_COLD, CoinFeature.EVM_WALLET];
 const OPETH_FEATURES = [...ETH_FEATURES, CoinFeature.MULTISIG_COLD, CoinFeature.EVM_WALLET];
+const ZKETH_FEATURES = [...ETH_FEATURES, CoinFeature.MULTISIG_COLD, CoinFeature.EVM_WALLET];
```

Comment:
We support multi sig cold for zketh? Not tss cold?

Reply:
Support for zkSync TSS wallets are not in the scope of the project. So we don't support